### DISCUSSION TITLE
SidesetInfoVectorPostprocessor with use_displaced_mesh=true

### DISCUSSION THREAD

**Discussion by brinkconti** [idaholab/moose#26775] [Q&A Modules: Solid mechanics] [answered]
Hi moose community,

I'm trying to get the bounding box of a sideset and found the "SidesetInfoVectorPostprocessor" which reports this as metadata. 

For the undeformed configuration is works, but when I set "use_displaced_mesh=true", than the debug version gives
```
Assertion `_boundary_data.find(_current_boundary_id) != _boundary_data.end()' failed
_current_boundary_id not found in _boundary_data.
at .../moose/framework/src/vectorpostprocessors/SidesetInfoVectorPostprocessor.C, line 82
```
and the opt version results in a core dump.

This file reproduces the issue:
[sideset_info.i.zip](https://github.com/idaholab/moose/files/14221260/sideset_info.i.zip)

Should this actually work or are there different ways to get a bounding box?

Thanks 
Maik

**Comment by GiudGiud** [comment]
Hello

The easiest way to get a bounding box for a sideset is to run the simulation with `--mesh-only` on the command line or with `show_info = true` in a mesh generator.
Of course neither of these work with displaced meshes, unless you reload the mesh in a FileMeshGenerator after having it displaced

I ll take a look at the file

Guillaume

**Comment by GiudGiud** [comment, marked as answer]
Thank you for reporting the issue. I put up a patch here
https://github.com/idaholab/moose/pull/26780

**Reply by brinkconti** [reply]
Thanks you for the quick response and solution!
